app/decorators/project_decorator.rb

```diff
@@ -0,0 +1,13 @@
+class ProjectDecorator < Draper::Decorator
+  delegate_all
+
+  # Define presentation-specific methods here. Helpers are accessed through
+  # `helpers` (aka `h`). You can override attributes, for example:
+  #
+  #   def created_at
+  #     helpers.content_tag :span, class: 'time' do
+  #       object.created_at.strftime("%a %m/%d/%y")
+  #     end
+  #   end
+
+end
```

Comment:
Remove commented code. If we need it we can look back into repo - this is what they are meant for.